frontend/include/chpl/resolution/scope-types.h

```diff
@@ -255,12 +305,12 @@ class BorrowedIdsWithName {
    */
   IdAndFlags::Flags filterFlags_ = 0;
   /**
-    Exclude this combination of flags; for example, if it is
+    Exclude these combinations of flags; for example, if one is
```

Comment:
might be good to move such examples to the FlagSet comment & show an example requiring or excluding when the list has two things.